src/storage/txn/actions/cleanup.rs

```diff
@@ -107,6 +110,61 @@ pub mod tests {
         cleanup(&mut txn, Key::from_raw(key), current_ts, true).unwrap_err()
     }
 
+    // Temporary test helper before implementing GC fence writing. After it's implemented, this
+    // should be replaced by writing a newer version, run cleanup, and then delete the newer write
+    // record.
```

Comment:
Is "implenting GC fence writing" not satisfied by #9222?

Reply:
It is. I'll update it.

Reply:
Fixed

Reply:
Comments aren't updated.

Reply:
Oh... sorry